breaking-changes.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.17
 
+- `javaScriptEnabledAndroid` of the `WebView` component has been renamed to `javaScriptEnabled`
```

Comment:
Won't it create confusion? No one is really using that prop as its been
added a few commits back -- though I agree it wont hurt :)
On Jan 5, 2016 7:31 PM, "Andy Street" <notifications@github.com> wrote:

> In breaking-changes.md
> <https://github.com/facebook/react-native/pull/5081#discussion_r48856039>:
>
> > @@ -2,6 +2,8 @@
> >
> >  ## 0.17
> >
> > +- `javaScriptEnabledAndroid` of the `WebView` component has been renamed to `javaScriptEnabled`
>
> Add that domStorageEnabledAndroid was also renamed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/react-native/pull/5081/files#r48856039>.
>